libs/partners/openai/langchain_openai/chat_models/azure.py

```diff
@@ -548,8 +549,10 @@ def validate_environment(cls, values: Dict) -> Dict:
         values["openai_api_key"] = (
             convert_to_secret_str(openai_api_key) if openai_api_key else None
         )
-        values["openai_api_base"] = values["openai_api_base"] or os.getenv(
-            "OPENAI_API_BASE"
+        values["openai_api_base"] = (
+            values["openai_api_base"]
+            if "openai_api_base" in values
+            else os.getenv("OPENAI_API_BASE")
```

Comment:
thanks! Just switched logic here to respect `openai_api_base=None` instead

Reply:
Nice! I like that logic better.